hooks/post_gen_project.py

```diff
@@ -41,6 +41,28 @@ def main() -> None:
     if "{{ cookiecutter.add_github_actions }}" == "no":
         remove_directory(project_path, ".github", "workflows")
 
+    if "{{ cookiecutter.add_python_example }}" == "no":
```

Comment:
Note to self: Will add a test similar to `test_check_conda_environment_creation_and_run_all_checks` where only R example is kept so to not introduce bugs based on the removal of Python files in the future.